src/base/base.service.ts

```diff
@@ -0,0 +1,15 @@
+import { BaseEntity, Repository } from 'typeorm';
+
+export default class BaseService<Entity extends BaseEntity> {
```

Comment:
I'm not sure `BaseService` is the correct name for this service, as it is 'base' only for repository services. Maybe we can use BaseRepositoryService or smth like that.

Reply:
FIxed